### ISSUE TITLE
Public URL don't seem to be working with SVG

### ISSUE THREAD

**Issue by grillorafael** [parcel-bundler/parcel#367]
**Choose one:** 🐛 bug report

### 🎛 Configuration (.babelrc, package.json, cli command)

```js
{
  "presets": ["react-app"]
}
```

```js
{
  "name": "<redacted>",
  "scripts": {
    "start": "cross-env NODE_ENV=development parcel index.html --public-url \"/v2/assets\" --out-dir ../public/v2",
    "build": "cross-env NODE_ENV=production parcel build index.html --public-url \"/v2/assets\" --out-dir ../public/v2",
    "clean": "rm -rf ./.cache ../public/v2"
  },
  "dependencies": {
    "autoprefixer": "^7.2.1",
    "babel-preset-react-app": "^3.1.0",
    "node-sass": "^4.7.2",
    "parcel-bundler": "^1.2.1",
    "react": "^16.2.0",
    "react-dom": "^16.2.0",
    "react-router-dom": "^4.2.2"
  },
  "postcss": {
    "modules": false,
    "plugins": {
      "autoprefixer": {
        "browsers": [
          ">1%",
          "last 4 versions",
          "Firefox ESR",
          "not ie < 9"
        ],
        "flexbox": "no-2009"
      }
    }
  },
  "devDependencies": {
    "cross-env": "^5.1.1"
  }
}

```

### 🤔 Expected Behavior
```
import React from 'react';
import { Link } from 'react-router-dom';
import logo from '../logo.svg';


const Home = () => {
    console.log(logo);
    return (
        <div className="App">
            <img className="App-Logo" src={logo} alt="React Logo" />
            <h1 className="App-Title">Hello Parcel x React</h1>
            <Link to="/other">Other</Link>
        </div>
    )
};

export default Home;
```

I expected src to be `/v2/assets/<svgpath>`

### 😯 Current Behavior

Gets rendered as `/v2/<svgpath>`

**Comment by grillorafael** [comment]
It does work when I put a trailing slash at the end of public url

**Comment by hufei1993** [comment]
I have the same question,

**Comment by hufei1993** [comment]
It seems that all the images do not work

**Comment by devongovett** [comment]
Should be fixed by #389 which was just merged. Will go out in the next release (shortly).

**Comment by ericaprieto** [comment]
Just tried the referenced PR and the problem seems to persist

**Comment by tscholl2** [comment]
I think I ran into this issue, or maybe a slightly related one? I have a smaller proof-of-concept though:

index.html:
```
<html><body><script src="./main.js"></script></body></html>
```
main.js:
```
const a = require("./a.svg");
const img = document.createElement("img");
img.src = a;
document.body.appendChild(img);
```
a.svg:
```
<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
  <circle cx="50" cy="50" r="50" fill="red"/>
</svg>
```

Running `parcel index.html` (which is using version 1.3.0), the page loads but appears blank. However, if I change line 3 in `main.js` to `img.src = "dist/" + a;`, then the red circle appears.

Is this related to this issue or am I just missing something?

**Comment by devongovett** [comment]
Should be fixed in v1.3.1.

**Comment by tscholl2** [comment]
I tested my example with 1.3.1 and it looks good! I got the opposite results, which I believe is the intended behavior. Thanks for the quick update!
